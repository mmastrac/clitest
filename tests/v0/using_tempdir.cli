#!/usr/bin/env clitest --v0

using tempdir;

defer {
    $ echo "cleaning up"
    ! cleaning up
}

using new dir "subdir";

$ touch foo

cd ..;

$ find .
! .
! ./subdir
! ./subdir/foo
