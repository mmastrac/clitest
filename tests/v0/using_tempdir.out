Running <root>/using_tempdir.cli ...

using tempdir: <tmp>

using new dir: <tmp>/subdir

touch foo
---
(no output)
---
✅ OK

cd ..

find .
---
.
./subdir
./subdir/foo
---
✅ OK

(cleanup) removing <tmp>/subdir && cd <tmp>

(deferred) echo "cleaning up"
---
cleaning up
---
✅ OK

(cleanup) removing <tmp> && cd <root>

<root>/using_tempdir.cli PASSED
